networkx/algorithms/centrality/betweenness_subset.py

```diff
@@ -187,11 +187,13 @@ def edge_betweenness_centrality_subset(G,sources,targets,
     return b
 
 # obsolete name
-def betweenness_centrality_source(G,normalized=True,weight=None,sources=None):
+def betweenness_centrality_source(G, normalized=True, weight=None,
+                                  sources=None):
     if sources is None:
-        sources=G.nodes()
-    targets=G.nodes()
-    return betweenness_centrality_subset(G,sources,targets,normalized,weight)
+        sources = list(G)
```

Comment:
@dschult targets can't be left as an iterator because `betweenness_centrality_subset` calls `_accumulate_subset` n times(no of nodes) and the iterator `targets` exhausts itself in the first iteration `target_set = set(targets)` and hence `target_set` is an empty set in the subsequent iterations.
One way to solve this is to declare it a set before entering the loop in `betweenness_centrality_subset`